package.json

```diff
@@ -45,33 +45,18 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/TryGhost/node-sqlite3.git"
+    "url": "https://github.com/microsoft/vscode-node-sqlite3.git"
   },
   "dependencies": {
-    "@mapbox/node-pre-gyp": "^1.0.0",
-    "node-addon-api": "^4.2.0",
-    "tar": "^6.1.11"
+    "node-addon-api": "^4.2.0"
```

Comment:
@deepak1556 is this version ok to use? I noticed `5.0.0` is out.

Reply:
We can update to the latest `5.0.0`. The current locked version was based on whatever was latest at that time.

Reply:
Oh nevermind, upstream is using the same version!